application/models/SurveyDynamic.php

```diff
@@ -404,7 +404,7 @@ public function getExtendedData($colName, $sLanguage, $base64jsonFieldMap)
                 $sSurveyEntry .= '<tr>';
                 if (isset($aFilesInfo[$iFileIndex])) {
                     $sSurveyEntry .= '<td>'.CHtml::link(CHtml::encode(rawurldecode($aFilesInfo[$iFileIndex]['name'])), App()->createUrl("/admin/responses", array("sa"=>"actionDownloadfile", "surveyid"=>self::$sid, "iResponseId"=>$this->id, "iQID"=>$oFieldMap->qid, "iIndex"=>$iFileIndex))).'</td>';
-                    $sSurveyEntry .= '<td>'.sprintf('%s Mb', round($aFilesInfo[$iFileIndex]['size'] / 1000, 2)).'</td>';
+                    $sSurveyEntry .= '<td>' . (($aFilesInfo[$iFileIndex]['size'] && is_float($aFilesInfo[$iFileIndex]['size'])) ? sprintf('%s Mb', round($aFilesInfo[$iFileIndex]['size'] / 1000, 2)) : "") . '</td>';
```

Comment:
Not too much inlove with inline-code, but that's a personal taste I guess, right?

Reply:
I do the quickest way here (3X). Maybe some uodate for 5 or 6 then ?

Reply:
See the previous celle (whu it's a cell ????)

`'<td>'.CHtml::link(CHtml::encode(rawurldecode($aFilesInfo[$iFileIndex]['name'])), App()->createUrl("/admin/responses", array("sa"=>"actionDownloadfile", "surveyid"=>self::$sid, "iResponseId"=>$this->id, "iQID"=>$oFieldMap->qid, "iIndex"=>$iFileIndex))).'</td>';`